alongtheway.txt

```diff
@@ -170,7 +170,7 @@ Is there anything God is asking you to give up?
 Day 5: after the mountain
 Read: Genesis 22:7-8, 13-18, Romans 4:1-5:12 (esp. 4:18-22) Hebrews 11:8-19 (esp. 17 and 18)
 
-Look back at a bit of Abraham’s journey. If you haven’t read the passage in Romans yet do so today. It and the passage in Hebrews give us a unique perspective on the story we have read this week.
+Look back at a bit of Abraham’s journey. Todays passages from Romans and Hebrews give us a unique perspective on the story we have read this week.
```

Comment:
clarity. I found the reference to the Romans passage on Day3 optional reading, but it's already in the 'READ' section for today..so, they'll read it. maybe I'll add a note in the 'READ' section